server/service/handler.go

```diff
@@ -506,6 +503,15 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	// Fleet Sandbox demo login (always errors unless config.server.sandbox_enabled is set)
 	ne.WithCustomMiddleware(limiter.Limit("login", throttled.RateQuota{MaxRate: loginRateLimit, MaxBurst: 9})).
 		POST("/api/_version_/fleet/demologin", makeDemologinEndpoint(config.Server.URLPrefix), demologinRequest{})
+
+	ne.WithCustomMiddleware(
+		errorLimiter.Limit("ping_device", desktopQuota),
+	).HEAD("/api/_version_/fleet/device_ping", devicePingEndpoint, devicePingRequest{})
+
+	ne.WithCustomMiddleware(
+		errorLimiter.Limit("ping_orbit", desktopQuota),
+	).HEAD("/api/_version_/fleet/orbit_ping", orbitPingEndpoint, orbitPingRequest{})
```

Comment:
Does `/api/fleet/orbit/ping` make more sense..? And his should probably not be versioned, right? Since we decided not to version `/orbit/enroll` and `/orbit/config` endpoints..?

Reply:
re: versioning I didn't know, thanks!

re: naming, the thing is we can't use `/device/ping` because it collides with `device/:token` so I went for consistency

Reply:
@sharvilshah do you remember why we decided to not version those endpoints? just trying to understand if it makes sense for the "ping" endpoints?

Reply:
I think it's because `osquery` endpoints are not versioned, and for consistency, @lucasmrod suggested here (https://github.com/fleetdm/fleet/pull/7246#discussion_r975440917) that`orbit` endpoints also should not be versioned

Reply:
gotcha, thanks! I'm inclined to leave this one versioned unless you or Lucas have any concerns

Reply:
Sounds good, I have no issues -- just something I noticed

Reply:
Versioning makes sense for the Fleet API, not sure it makes sense for these internal endpoints that we plan to keep stable.

Reply:
Forgot to mention we should tackle this before merge. We can tackle it in the other PR though.

Osquery and Fleet Desktop APIs are not versioned
Only Fleet user API is versioned (because it's made to be consumed by users)
IMO Orbit APIs should not be versioned (they are of the same type as Osquery's and Fleet Desktop's)

Reply:
ahh gotcha, I understood the opposite: that the ping endpoints were considered part of the fleet API.

I will send a PR in short, thanks both!

Reply:
two notes:

1. Fleet Desktop APIs _are_ currently versioned, not sure if we can change this now.
2. Given that we wont have collisions anymore, I will use `/ping/orbit` and `/ping/desktop` as Sharvil suggested

Reply:
(1) Ouch, mhm... I missed this. Why are they versioned?...

@chiiph Do you remember why Fleet Desktop APIs are versioned? IMO they shouldn't.

Reply:
PR with the changes: https://github.com/fleetdm/fleet/pull/7933/files

Reply:
> @chiiph Do you remember why Fleet Desktop APIs are versioned? IMO they shouldn't.

They might've just fallen with the rest. I guess in here we have capabilities, so versioning is not needed.